nixos/cosmic/module.nix

```
@@ -61,6 +61,7 @@ in
       hicolor-icon-theme
       pop-icon-theme
       pop-launcher
+      playerctl
```

```suggestion
      playerctl
      pop-icon-theme
      pop-launcher
```

Can you lexicographically sort these?

---

Done! Sorry I had not noticed it was sorted, as everything started with `cosmic` x)